tests/functional/fork_detected.ts

```diff
@@ -0,0 +1,60 @@
+/* tslint:disable:no-relative-imports */
+import { describe } from 'riteway'
+
+import { bitcoindClients, resetBitcoinServers, generateReorgs } from '../helpers/bitcoin'
+import { delay, runtimeId } from '../helpers/utils'
+import { setUpServerAndDb } from '../helpers/utils'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+
+const blockchainSettings = {
+  MINIMUM_BLOCK_HEIGHT: 0,
+  ENABLE_ANCHORING: true,
+  ANCHOR_INTERVAL_IN_SECONDS: 10,
+  BATCH_CREATION_INTERVAL_IN_SECONDS: 5,
+  READ_DIRECTORY_INTERVAL_IN_SECONDS: 5,
+  UPLOAD_CLAIM_INTERVAL_IN_SECONDS: 5,
+}
+
+const { bitcoinCoreClientA, bitcoinCoreClientB }: any = bitcoindClients()
+
+describe('Fork detected', async (assert: any) => {
+  await resetBitcoinServers()
+  await bitcoinCoreClientB.addNode(bitcoinCoreClientA.host, 'add')
+  await delay(5 * 1000)
+
+  const { db, server, rabbitMQ } = await setUpServerAndDb({ PREFIX, NODE_PORT, blockchainSettings })
+
+  await delay(5 * 1000)
+
+  const FORK_DETECTED = `${PREFIX}.FORK_DETECTED`
+  await generateReorgs(bitcoinCoreClientA, bitcoinCoreClientB)
+
+  const getFork = (): Promise<{ blockHash: string, blockHeight: number }> => new Promise(async (resolve) => {
```

Comment:
What happens if the RMQ message never is published? Does the test timeout, or does it hang forever?